controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -257,10 +257,17 @@ func (r *DataScienceClusterReconciler) Reconcile(ctx context.Context, req ctrl.R
 	}
 
 	// finalize reconciliation
+	operatorReleaseVersion, err := cluster.SetRelease(r.Client)
+	if err != nil {
+		r.Log.Error(err, "failed to get operator release version")
+		return ctrl.Result{}, err
```

Comment:
Do we consider it as a critical error?

Reply:
We probably need this to be critical error. It points to either error getting platform or CSV